src/Elements/StyledSelect.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react'
+
+class StyledSelect extends Component {
```

Comment:
If the only thing this class adds is the default styling, then we might as well put the select style in the global style or in the stylesheet for the settings screen and use a normal `<select ...>`.

I think I recommended making it a component before - my thought was that the component should handle enumerating interfaces, and displaying those.

Reply:
That said, I wonder if we should enumerate network interfaces in the renderer process. If we want to keep the web interface portable, it's probably best if this happens in the main process.

Reply:
The `<select>` tag needs to be wrapped in a div that gets styled. The purpose of this component is to make it so I don't have to wrap every select in a div for styling purposes.

Reply:
To be generic, this component **should not** enumerate the interfaces. If you want, I can create a component `NetworkInterfacesSelect` that uses this component which just does the enumeration.

Reply:
re enumerating in the renderer process vs the main process, the rule of thumb we should follow is to only do browser-compatible things in the renderer process. Since this code wouldn't run in a browser it should be moved.